data/json/vehicle_parts.json

```diff
@@ -3601,7 +3601,7 @@
         "range" : 12,
         "fuel_type" : "plasma",
         "item" : "plasma_rifle",
-        "difficulty" : 7,
+        "difficulty" : 6,
```

Comment:
There's laser turrets, plasma turrets, and fusion turrets.  They were all at 7, but I lowered the plasma turret to 6 to match the plasma engine, and the laser turret to match the plasma turret.  Which of the three is the most exoticest?